src/promptflow-devkit/promptflow/_sdk/_service/utils/utils.py

```diff
@@ -125,10 +126,15 @@ def is_port_in_use(port: int):
         return s.connect_ex(("localhost", port)) == 0
 
 
-def get_random_port():
-    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
-        s.bind(("localhost", 0))
-        return s.getsockname()[1]
+def get_pfs_port():
+    port = 23333
```

Comment:
: ) I think we should add some docs on this constant port in our pf service start help

Reply:
added